src/Components/HomePage/HomePage.js

```diff
@@ -1,11 +1,44 @@
+import { useContext } from 'react'
+import { UserContext } from '../../Providers/UserContext'
+
 import TimelinePost from "../TimelinePost/TimelinePost"
 import "./_HomePage.scss"
 
+//////////// TO DO LIST ////////////
+//1. Bookmark button
+//1. Timeline segment logic
+//1. Reposition timeline circle, line, and recommender avatar lower
+
 const HomePage = () => {
+  const { currentUser } = useContext(UserContext)
+  console.log("HERE: ", currentUser.recommendations[0])
+
+  const sortedList = currentUser.recommendations.sort((a, b) => {
```

Comment:
Great use of .sort() to sort the recommendations by date (most recent to least recent)!

Reply:
Appreciate the comments about Context and passing Props. I am utilizing Context to access our data, however I still need to pass props when I iterate through the lister and pass extracted information to child components. I am not sure of a way to strictly use Context when it comes to iterating through a list to create a component, but let me know if you have a better application of this

Reply:
Hmmm I think passing props is acceptable in the context. I think you are on the right track after we had the Context API discussion this morning.